core/src/components/menu/menu.tsx

```diff
@@ -396,6 +396,28 @@ export class Menu implements ComponentInterface, MenuI {
     }
 
     this.beforeAnimation(shouldOpen);
+
+    if (shouldOpen) {
+      // focus menu content for screen readers, preserving scroll position
+      if (this.menuInnerEl) {
```

Comment:
I know Sean had suggested focusing the host of `ion-menu` instead of trying to mess with the scroll position. Any reason why we did not do that here?

Reply:
Another thought: We do have an open issue with focus + menu (FW-293). Perhaps focusing the `ion-menu` host would help resolve that issue as well?

You could do something like the following:

```
// check to see if the activeElement is inside of ion-menu
  // if it is, do nothing and return early
// otherwise focus ion-menu
```

That way you can get the benefits of focus trapping without having to manage the scroll position.

Reply:
I thought it would cause screen reader issues, but I gave it a try and it seems to work okay 🤔 I'll fiddle with it some more in the morning.

Reply:
Tried it with just focusing `this.el` instead of the first descendant (and leaving focus trapping alone) and that seems to work great 😆 No scroll jumping, and it plays nice with screen readers.